lib/active_model/serializer/attributes.rb

```diff
@@ -0,0 +1,59 @@
+module ActiveModel
+  class Serializer
+    # Defines the attributes to be serialized.
+    #
+    module Attributes
```

Comment:
This module is directly extracted from the serializer, no modifications to the code have been made, except for:
 1. the definition of an `inherit_attributes` method, rather than overriding `inherited` as in the `Associations` module, which made it impossible for the two modules to define the inherited properties
 2. the optional block parameter to the `attribute` method.